github/Commit.py

```diff
@@ -256,3 +265,5 @@ def _useAttributes(self, attributes):
             self._stats = self._makeClassAttribute(github.CommitStats.CommitStats, attributes["stats"])
         if "url" in attributes:  # pragma no branch
             self._url = self._makeStringAttribute(attributes["url"])
+        if "tree" in attributes:  # pragma no branch
+            self._tree = self._makeClassAttribute(Commit, attributes["tree"])
```

Comment:
This doesn't really make sense. The tree is not a commit. The JSON also doesn't quite provide enough information to construct a `GitTree` object either (it's missing the subtree elements).

It might make more sense to expose a property called `tree_sha` instead. Then, it could be used in conjunction with `Repository.get_git_tree()`.

Reply:
Seems like I was confusing the Commit with GitCommit, actually the tree attribute is already. So I will just add the missing unit test for this PR.